build/repo.targets

```diff
@@ -55,4 +56,20 @@
     <RemoveDir Directories="$(ProductPackagesSource)" />
   </Target>
 
+  <Target Name="PublishNpmModules">
+    <PropertyGroup>
+      <AuthTokenSetting>$(NpmRegistry.Replace("https:", "")):_authToken</AuthTokenSetting>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <NpmModule Include="$(ArtifactsDir)build\*.tgz" />
+    </ItemGroup>
+    
+    <Exec Command="npm config set $(AuthTokenSetting) $(APIKEY)" />
+    <Exec Command="npm publish --registry $(NpmRegistry) %(NpmModule.Identity)" ContinueOnError="true">
```

Comment:
I want to revert changes to the config regardless of whether publishing packages was successful or not to not leave the api key